ykbh/src/lib.rs

```diff
@@ -5,36 +5,37 @@ use ykpack::{
     UnsignedInt,
 };
 use yktrace::sir::SIR;
-use std::mem::size_of;
-
-const ALIGN: usize = size_of::<usize>();
 
 pub struct SIRInterpreter {
     /// Pointer to allocated memory containing a frame's locals.
     locals: *mut u8,
     /// The offset of each Local into locals.
     offsets: Vec<usize>,
+    /// The layout of locals. Needed for deallocating locals upon drop.
+    layout: Layout,
 }
 
 impl SIRInterpreter {
     pub fn new(local_decls: &Vec<LocalDecl>) -> Self {
         // FIXME Soon this will be pre-computed and handed to us by SIR.
-        let mut size: usize = 0;
         let mut offsets = Vec::new();
+        let mut layout = Layout::from_size_align(0, 1).unwrap();
```

Comment:
This feels a bit evil: would `Layout::new::<u8>()` be a bit clearer as I *think* that's what you're emulating here.

Reply:
I have no preference. I got this line from one of the examples in the docs: https://doc.rust-lang.org/std/alloc/struct.Layout.html#examples

Reply:
Let's use `unwrap`.

Reply:
You mean use `Layout::new::<u8>()`?

Reply:
Mea culpa, that'll teach me not to reload the GitHub tab! I see what you mean now. We can keep it as-is.